AUTHORS

```diff
@@ -1,44 +1,67 @@
 Karl Kosack <karl.kosack@cea.fr>
-Jason Watson <jason.watson@physics.ox.ac.uk>
+Maximilian Linhoff <maximilian.linhoff@tu-dortmund.de>
+Jason Watson <jason.watson@desy.de>
 Daniel Parsons <dparsons@lfs2.mpi-hd.mpg.de>
 Jean Jacquemier <jacquem@lapp.in2p3.fr>
 Tino Michael <tino.michael@cea.fr>
-Maximilian Nöthe <maximilian.noethe@tu-dortmund.de>
+Lukas Nickel <lukas.nickel@tu-dortmund.de>
+Noah Biederbeck <noah.biederbeck@tu-dortmund.de>
+Tomas Bylund <tomas.bylund@cea.fr>
+Lukas Beiske <lukas.beiske@tu-dortmund.de>
+Georg Schwefer <georg.schwefer@mpi-hd.mpg.de>
+Jonas Hackfeld <jonas.hackfeld@ruhr-uni-bochum.de>
 Alison Mitchell <alison.mitchell@mpi-hd.mpg.de>
 Dominik Neise <neised@phys.ethz.ch>
+Michele Peresano <peresano.michele@gmail.com>
 Christoph Deil <Deil.Christoph@gmail.com>
 Justus Zorn <justus.zorn@desy.de>
+Anno Knierim <anno.knierim@tu-dortmund.de>
+Stefan Fröse <stefan.froese@udo.edu>
+Rune Michael Dominik <rune.dominik@tu-dortmund.de>
 Samuel Timothy Spencer <sojourner92@gmail.com>
 Raquel de los Reyes <reyes@mpi-hd.mpg.de>
 Franca Cassol <cassol@cppm.in2p3.fr>
+Clara Escanuela Nieves <clara.escanuela@mpi-hd.mpg.de>
+Thomas Vuillaume <thomas.vuillaume@lapp.in2p3.fr>
 Kai Brügge <kai.bruegge@tu-dortmund.de>
+Felix Werner <felix.werner@mpi-hd.mpg.de>
+Satoshi Fukami <fukami@icrr.u-tokyo.ac.jp>
 Michele Mastropietro <michele.mastropietro@gmail.com>
 Jeremie DECOCK <jd.jdhp@gmail.com>
+Abelardo Moralejo Olaizola <moralejo@ifae.es>
+Mykhailo Dalchenko <mykhailo.dalchenko@cern.ch>
 Tarek Hassan <tarek.hassan@desy.de>
 Wrijupan Bhattacharyya <wrijupan.bhattacharyya@desy.de>
 Francesco Visconti <france.visconti@gmail.com>
-Abelardo Moralejo Olaizola <moralejo@ifae.es>
-Markus Gaug <markusgaug@gmail.com>
+Vadym Voitsekhovskyi <vadym.voitsekhovskyi@unige.ch>
 Jose Enrique Ruiz <jer@iaa.es>
+Markus Gaug <markusgaug@gmail.com>
 Paolo Cumani <pcumani@ifae.es>
 Thomas Gasparetto <thomas.gasparetto@ts.infn.it>
 Ruben Lopez-Coto <rlopezcoto@gmail.com>
-Lukas Nickel <lukas.nickel@tu-dortmund.de>
-Thomas Vuillaume <thomas.vuillaume@lapp.in2p3.fr>
+Orel Gueta <orel.gueta@desy.de>
 Tristan Carel <tristan.carel@lapp.in2p3.fr>
-Miguel Nievas <miguelnievas@ucm.es>
-Michael Punch <michael.punch@lnu.se>
 Daniel Nieto <nieto@gae.ucm.es>
-Orel Gueta <orel.gueta@desy.de>
-Julien Lefaucheur <jjlk@users.noreply.github.com>
-Thomas Armstrong <thomas.armstrong@physics.ox.ac.uk>
+Michael Punch <michael.punch@lnu.se>
```

Comment:
I suspect @mdpunch  would prefer to use punch@in2p3.fr

Reply:
That's the address in his commiter info, I can add an entry to the mailmap if michael wants

Reply:
Thanks @Tobychev , indeed I prefer the punch@in2p3.fr one, since I don't have the LnU one anymore!

I will see how to change my committer info, but in the meantime please use the in2p3 address.

Reply:
I added an entry to the mailmap for you

Reply:
Thanks, I updated my commit address now (but it's not retroactive).